### ISSUE TITLE
UGens: Range of PinkNoise is between -0.6 and 0.6

### ISSUE THREAD

**Issue by LFSaw** [supercollider/supercollider#2678]
Is there a reason why PinkNoise ranges between -0.4 and 0.4?
I thought it has to do with having equal power than WhitenNoise but this is not so... (it has actually way less).

I think, this needs documentation (since it is difficult to "fix").

```
// this uses my implementation of RMS implementing 
// rms = sqrt(lpf1(x^2))
// available in the DEIND UGens within SC3 Plugins

{RMS.ar([PinkNoise.ar, WhiteNoise.ar])}.plot(1)
```

![screen shot 2017-01-27 at 18 35 09](https://cloud.githubusercontent.com/assets/75468/22380848/5d4f7dee-e4bf-11e6-83d3-71c04b64350c.jpg)

**Comment by Sciss** [comment]
This is from my own docs:

> The values produced by this UGen were observed to lie with very high probability between approx. -0.65 and +0.81 (before being multiplied by mul ). The RMS is approx. -16 dB.

So I guess with very small likelihood, you'll hit the -1, +1 boundaries perhaps

**Comment by LFSaw** [comment]
Thanks, @Sciss . I'd suggest to add these words to the PinkNoise helpfile.

**Comment by jamshark70** [comment]
Pink noise is "1/f noise." Digitally, that's often calculated by the Voss algorithm, which is effectively an average of multiple random number generators running at different rates [1]. The distribution of an average of random numbers will be weighted toward the center, so, if you're generating pink noise the Voss way, it's theoretically possible to hit the extremes of the range, but unlikely.

If you search, you'll also find references to the Voss-McCartney algorithm -- that's "our" McCartney :) who figured out that you can offset the random number generators so that you're only ever evaluating one of them per sample (whereas the Voss algorithm lines them up).

[1] http://www.firstpr.com.au/dsp/pink-noise/#Voss - "the Voss algorithm, which creates pink noise by adding a series of white noise sources at successively lower octaves"

**Comment by telephon** [comment]
See also http://swiki.hfbk-hamburg.de/MusicTechnology/738

**Comment by LFSaw** [comment]
So, what to do with this? should we mention it in the helpfile?

**Comment by telephon** [comment]
Should I move the examples to the `examples/` folder and then we can reference them from the hepfile?

**Comment by LFSaw** [comment]
that'd be super, @telephon . I can also add them to my [topic/pinknoise-help](https://github.com/supercollider/supercollider/compare/master...LFSaw:topic/pinknoise-help) branch.

**Comment by brianlheim** [comment]
Fixed via #2689.
